cms/standard_pages/models.py

```diff
@@ -38,3 +40,76 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage", "InformationPage"]
+    subpage_types: ClassVar[list[str]] = ["IndexPage", "InformationPage"]
+
+    description = models.TextField(blank=True)
+    featured_pages = StreamField([("featured_page", RelatedContentBlock())], blank=True)
+    content = RichTextField(features=settings.RICH_TEXT_BASIC, blank=True)
+    related_links = StreamField([("related_link", RelatedContentBlock())], blank=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("description"),
+        FieldPanel("featured_pages"),
+        FieldPanel("content"),
+        FieldPanel("related_links"),
+    ]
+
+    search_fields: ClassVar[list[index.SearchField | index.AutocompleteField]] = [
+        *BasePage.search_fields,
+        index.SearchField("description"),
+    ]
+
+    def get_formatted_featured_pages(self) -> list[dict[str, str]]:
+        formatted_featured_pages = []
+
+        if featured_pages := self.featured_pages:
+            for featured_page in featured_pages:
+                formatted_featured_pages.append(
+                    {
+                        "featured": "true",
+                        "title": {"text": featured_page.value.link["text"], "url": featured_page.value.link["url"]},
+                        "description": featured_page.value["description"],
+                    }
+                )
+
+        elif child_pages := self.get_children().live().public():
+            for child_page in child_pages:
+                formatted_featured_pages.append(
+                    {
+                        "featured": "true",
+                        "title": {"text": child_page.title, "url": child_page.url},
+                    }
+                )
+
+        return formatted_featured_pages
+
+    def get_formatted_related_links_list(self) -> list[dict[str, str]]:
+        """Returns a formatted list of related links for both external and internal pages
+        for use with the Design System list component.
+        """
+        formatted_links = []
+
+        for related_link in self.related_links:  # pylint: disable=not-an-iterable
+            formatted_links.append(
+                {
+                    "title": related_link.value.link.get("text"),
+                    "url": related_link.value.link.get("url"),
+                }
+            )
+
+        return formatted_links
+
+    def get_context(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> dict:
+        context: dict = super().get_context(request, *args, **kwargs)
+
+        context["formatted_featured_pages"] = self.get_formatted_featured_pages()
```

Comment:
```suggestion
        context["formatted_items"] = self.get_formatted_items(request)
```

Reply:
Done in [Improvements in get_formatted_items()](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/793e6bff59f48cf85f7698649e014aeb2f6074a7)